python/lsst/sims/maf/metrics/calibrationMetrics.py

```diff
@@ -227,6 +227,11 @@ def __init__(self, metricName='ParallaxCoverageMetric', m5Col='fiveSigmaDepth',
         else:
             self.mags = utils.stellarMags(SedTemplate, rmag=rmag)
         self.atm_err = atm_err
```

Comment:
On line 185 in the docstring separated is spelled wrong. 
On line 186 in the specific is spelled wrong.
On line 188 observations is spelled wrong.